content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -588,3 +597,299 @@ As part of the Peruvian localization, besides creating credit notes from an exis
+
+The *carrier* is used when the delivery guide is through *public* transport.
+
+To create a new carrier, navigate to :menuselection:`Contacts --> Create` and fill out the contact
+information.
+
+First, select :guilabel:`Company` as the :guilabel:`Company Type`. Then, add the :guilabel:`MTC
+Registration Number`, :guilabel:`Authorization Issuing Entity`, and the :guilabel:`Authorization
+Number`.
+
+For the company address, make sure the following fields are complete:
+
+- :guilabel:`District`
+- :guilabel:`Tax ID` (:guilabel:`DNI`/:guilabel:`RUC`)
+- :guilabel:`Tax ID Number`
+
+.. image:: peru/company-operator-configuration.png
+   :align: center
+   :alt:  Company type operator configurations in the Contact form.
+
+Vehicles
+********
+
+To configure the available vehicles, navigate to :menuselection:`Inventory --> Configuration -->
+Vehicles` and fill in the vehicle form with the information needed for the vehicle:
+
+- :guilabel:`Vehicle Name`
+- :guilabel:`License Plate`
+- :guilabel:`Is M1 or L?`
+- :guilabel:`Special Authorization Issuing Entity`
+- :guilabel:`Authorization Number`
+- :guilabel:`Default Operator`
+- :guilabel:`Company`
+
+.. important::
+   It is important to check the :guilabel:`Is M1 or L?` checkbox if the vehicle has fewer than four
+   wheels or fewer than eight seats.
+
+.. image:: peru/vehicle-not-m1-or-l-pe.png
+   :align: center
+   :alt: Vehicle not selected as an M1 or L type with extra fields shown.
+
+Products
+********
+
+To configure the available products, navigate to :menuselection:`Inventory --> Products` and open
+the product to be configured.
+
+Make sure that the applicable information in the product form is fully configured. The
+:guilabel:`Partida Arancelaria` (Tariff Item) field needs to be completed.
+
+Generating a GRE
+~~~~~~~~~~~~~~~~
+
+Once the delivery from inventory is created during the sales workflow, make sure you complete the
+|GRE| fields on the top-right section of the transfer form for the fields:
+
+- :guilabel:`Transport Type`
+- :guilabel:`Reason for Transfer`
+- :guilabel:`Departure start date`
+
+It is also required to complete the :guilabel:`Vehicle` and :guilabel:`Operator` fields under the
+:guilabel:`Guia de Remision PE` tab.
+
+The delivery transfer has to be marked as *Done* for the :guilabel:`Generar Guia de Remision` button
+to appear on the left menu of the transfer form.
+
+.. image:: peru/generate-gre-transferview.png
+   :align: center
+   :alt: Generar Guia de Remision button on a transfer form in the Done stage.
+
+Once the transfer form is correctly validated by |SUNAT|, you will see the XML file generated in the
+chatter. You can now print the delivery slip that will show the transfer details and the QR code
+validated by |SUNAT|.
+
+.. image:: peru/gre-delivery-slip.png
+   :align: center
+   :alt: Transfer details and QR code on generated delivery slip.
+
+Common errors
+~~~~~~~~~~~~~
+
+- `Diferente prefijo para productos (T001 en algunos, T002 en otros)`
```

Comment:
I guess such errors happen on dbs set in Spanish but for consistency, I'd write everything in English even though it will get translated later.
Though you might have some other reason behind that, let me know

Reply:
Hello @chiaraprattico @samueljlieber We documented them in Spanish because nevertheless those errors are sent from SUNAT (the fiscal entity in Peru) and the errors will always show up in spanish in the DB (even if the DB is in another language). I think this adds more value in spanish, since they could search the documentation just as the error appears in the DB. Let me know what you think!